### DISCUSSION TITLE
Strimzi 0.32.0 - kafka , zookeeper and schema-registry version ?

### DISCUSSION THREAD

**Discussion by userx91** [strimzi/strimzi-kafka-operator#8581] [Q&A] [answered]
Hi,
we are using strimzi 0.32.0.
From release page (https://github.com/strimzi/strimzi-kafka-operator/releases?page=2) I see it supports kafka version 3.3.1 . 
But is there a way to find the kafka version , zookeeper version and the schema register version that is used under the hood for version 0.32.0 ?

**Comment by scholzj** [comment]
Strimzi is based on Apache Kafka. So when you use Apache Kafka 3.3.1, it will use the ZooKeeper version which Apache Kafka 3.3.1 uses. I think it will be ZooKeeper 3.6.3, but you can doublecheck in the Kafka releases.

There is no schema registry as pat of Apache Kafka or Strimzi.

**Reply by scholzj** [reply]
BTW: The full list of supported Kafka versions in each Strimzi release is on the website: https://strimzi.io/downloads/

**Reply by userx91** [reply]
From the list of supported kafka version for 0.32.0 , how do we confirm the kafka version that is exactly used within 0.32.0 ?

**Reply by userx91** [reply]
and yes you are right kafka 3.3.1 uses version 3.6.3 for zookeper

**Reply by scholzj** [reply]
I'm not sure I follow -> when it supports some Kafka version then it means that this is a Kafka version that you can deploy.

**Reply by userx91** [reply]
Sorry , what I meant was , the downloads page that you have provided says the supported versions of kafka for 0.32.0 is 3.2.0, 3.2.1, 3.2.3, 3.3.1 . I wanted to know exactly what version of kafka is running bundled with it from this supported list of kafka versions ? 
Also does strimiz uses confluent kafka underneath ?

**Reply by scholzj** [reply, marked as answer]
> Also does strimiz uses confluent kafka underneath ?

Strimzi is an open-source project - and as such we use the Apache Kafka open-source project and its binaries. 

> Sorry , what I meant was , the downloads page that you have provided says the supported versions of kafka for 0.32.0 is 3.2.0, 3.2.1, 3.2.3, 3.3.1 . I wanted to know exactly what version of kafka is running bundled with it from this supported list of kafka versions ?

These are exactly the versions that it will be running. You can specify the exact version out of these in the custom resource when deploying the cluster and this version will be deployed. Or it will use the latest when you don't specify an exact version you want to use.

**Reply by userx91** [reply]
Thank you for the clear reply . I am looking for docker image that would tie in with the apachae kafka . I see confluent images for kafka , bitnami images for kafka , but I am not able to find the apachae kafka docker images . Although I see few package providers for apache kafka , I am not sure if I would be able to recreate the same infra , if I use those images . Hence to mimic the strimzi ecosystem , would you be kind enough to recommend docker image for "apache-kafka" ?

**Reply by scholzj** [reply]
I'm afraid I do not have any recommendations. The Strimzi images are basically designed to be used with Strimzi. You can use them to some extent outside of Strimzi as well - such as here: https://github.com/scholzj/strimzi-compose-up. But you would probably run into many limitations if you try to use them seriously that way.

I know that some people use the Bitnami images, some use also the [Wurstmeister images](https://github.com/wurstmeister/kafka-docker). But I'm afraid that I do not have any personal experience as I use Strimzi. One thing to keep in mind is that Confluent Platform is not exactly the same as Apache Kafka - as far as I know it has some changes and private features. I'm also not sure about their licensing. So that is something to keep in mind.
